clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -118,6 +118,15 @@ func NewPrometheusController(clusterLoaderConfig *config.ClusterLoaderConfig) (p
 		clusterLoaderConfig.PrometheusConfig.ScrapeKubeProxy = mapping["PROMETHEUS_SCRAPE_KUBE_PROXY"].(bool)
 	}
 	mapping["PROMETHEUS_SCRAPE_KUBELETS"] = clusterLoaderConfig.PrometheusConfig.ScrapeKubelets
+
+	// If enabled prometheus scrape windows node, need to setup windows node and template mapping
+	if enableScrapingWindowsNode(mapping, clusterLoaderConfig) {
```

Comment:
Can we move it outside the constructor? I believe a better place for that code would be in the SetUpPrometheusStack method

Reply:
Agree, moved